.github/workflows/upload_component.yml

```diff
@@ -61,5 +61,6 @@ jobs:
             usb/usb_host_vcp;
             zlib;
             libpng;
+            freetype;
```

Comment:
Since this component doesn't depend on other components, could you add it to the list while preserving the alphabetical order? See the comment at the top of the list for details.